openmdao/drivers/scipy_optimizer.py

```diff
@@ -258,11 +258,11 @@ def run(self):
         self.opt_settings['disp'] = self.options['disp']
 
         # Size Problem
-        nparam = 0
-        for param in self._designvars.values():
-            size = param['global_size'] if param['distributed'] else param['size']
-            nparam += size
-        x_init = np.empty(nparam)
+        ninput = 0
+        for input in self._designvars.values():
+            size = input['global_size'] if input['distributed'] else input['size']
+            ninput += size
+        x_init = np.empty(ninput)
```

Comment:
to be technical, let's replace these with 'ndesvar' and 'desvar' instead of input.